keys_storage.go

```diff
@@ -23,10 +31,10 @@ type KeysStorage interface {
 
 // KeysStorageInMemory is an in-memory message keys storage.
```

Comment:
Maybe worth expanding the comment to cover what the `uint` represents here (the message number) without needing to refer to the code below?